### PR TITLE
Clt ols combo

### PR DESCRIPTION
**Pull request by nbuonin**

### CODE REVIEW

File: src/clt_least_squares/InputForm.jsx [hunk 1]
```diff
@@ -17,191 +17,192 @@ export const InputForm = ({seed, handleSeed, handleGeneratePop, beta,
...
-                        <div style={{ height: '50px', width: '100%'}}>
-                            <Rheostat
-                                min={-100}
-                                max={100}
-                                values={[math.round(beta * 100, 2)]}
-                                onValuesUpdated={(sliderState) => {
-                                    handleBeta(
-                                        (sliderState.values[0] * 0.01));
-                                }} />
+        <div className={'sticky-top'}>
```
Comment (sdreher) [inline src/clt_least_squares/InputForm.jsx]: The sticky scroll is only working for me at certain points. I can review this with you tomorrow.

File: src/clt_least_squares/CLTLeastSquares.jsx [hunk 2]
```diff
@@ -110,6 +112,14 @@ export class CLTLeastSquares extends Component {
             return jStat.variance(residuals) * (99 / 98);
         });
     }
+    getPopulationSSE(population, populationRegression) {
```
Comment (sdreher) [inline src/clt_least_squares/CLTLeastSquares.jsx]: good candidate for a test? (I might just not be seeing where the test is btw)

File: src/clt_least_squares/CLTLeastSquares.jsx [hunk 3]
```diff
@@ -159,7 +159,7 @@ export class CLTLeastSquares extends Component {
             <>
             <Nav />
             <div className={'container'}>
-                <h2>Central Limit - Least Squares</h2>
+                <h2>Sampling Distribution of Regression Coefficients</h2>
```
[the remark this answers is not in the document]
Comment (nbuonin) [reply src/clt_least_squares/CLTLeastSquares.jsx]: Ach, yes, it should match the homepage.

File: src/clt_least_squares/InputForm.jsx (hunk 1 above)
Comment (nbuonin) [reply src/clt_least_squares/InputForm.jsx]: Ok cool.  This sticky scroll has caused me some agita - it doesn't work at all on Safari.  Lets look together.

File: src/clt_least_squares/CLTLeastSquares.jsx (hunk 2 above)
Comment (nbuonin) [reply src/clt_least_squares/CLTLeastSquares.jsx]: The real meat of this is in `calculateSSE`. I don't have a unit test for that, but I do call it in tests for the Least Squares sim.
In any case, this method, and `findLinearRegression` should have proper unit tests on them.

File: src/clt_least_squares/CLTLeastSquares.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (nbuonin) [reply src/clt_least_squares/CLTLeastSquares.jsx]: Or actually, I'm not sure. The one on the homepage is good, but let me check with the rest of the team.

**Pull request merged**